deepchem/models/autoencoder_models/autoencoder.py

```diff
@@ -0,0 +1,134 @@
+"""
+TODO(LESWING) Remove h5py dependency
+TODO(LESWING) Remove keras dependency and replace with functional keras API
+"""
+
+from deepchem.models import Model
+from deepchem.models.autoencoder_models.model import MoleculeVAE
+from deepchem.feat.one_hot import zinc_charset
+import os
+from subprocess import call
+
+
+class TensorflowMoleculeEncoder(Model):
```

Comment:
Could you add a docstring here explaining what the class does and providing a short usage example?